easybuild/main.py

```diff
@@ -223,8 +223,6 @@ def main(args=None, logfile=None, do_build=None, testing=False):
         # print location to last log file, and exit
         last_log = find_last_log(logfile) or '(none)'
         print_msg(last_log, log=_log, prefix=False)
-        cleanup(logfile, eb_tmpdir, testing, silent=True)
```

Comment:
This should be fine, there's lots of weird combo's of options you can make now.

Using `--last-log` and `--search` together won't break anything (I checked).